scripts/motor_diagram_data/connect-escs-and-motors.rst

```diff
@@ -19,163 +19,15 @@ Motor order diagrams
-    :target: ../_images/motororder-quad-v-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-quad-h-2d.png
-    :target: ../_images/motororder-quad-h-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-quad-vtail-2d.png
-    :target: ../_images/motororder-quad-vtail-2d.png
-    :scale: 35%
-
-.. image:: ../images/motororder-quad-atail-2d.png
-    :target: ../_images/motororder-quad-atail-2d.png
-    :scale: 35%
-
-.. image:: ../../../images/motororder-Y4a-2d.jpg
-    :target: ../_images/motororder-Y4a-2d.jpg
-    :scale: 40%
-
-Tricopter
----------
-
-.. image:: ../images/motororder-tri-a-2d.png
-    :target: ../_images/motororder-tri-a-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-tri-b-2d.png
-    :target: ../_images/motororder-tri-b-2d.png
-    :scale: 40%
-
-.. note::
-
-   If the direction of your tail servo is going the wrong way in
-   response to yaw then either the ``RCn_REVERSE`` RC input direction or the tilt servo's ``SERVOn_REVERSE`` parameter should be set to 1 (from 0), See :ref:`TriCopter setup page <tricopter>` for
-   details.)
-
-Bicopter
----------
-
-.. image:: ../images/motororder-bicopter.png
-    :target: ../_images/motororder-bicopter.png
-    :scale: 40%
-
-Hexacopter
-----------
-
-.. image:: ../images/motororder-hexa-x-2d.png
-    :target: ../_images/motororder-hexa-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-hexa-cw-x-2d.png
-    :target: ../_images/motororder-hexa-cw-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-hexa-plus-2d.png
-    :target: ../_images/motororder-hexa-plus-2d.png
-    :scale: 40%
-
-Y6
---
-
-.. image:: ../images/motororder-y6a-2d.png
-    :target: ../_images/motororder-y6a-2d.png
-    :scale: 35%
-
-.. image:: ../images/motororder-y6b-2d.png
-    :target: ../_images/motororder-y6b-2d.png
-    :scale: 35%
-
-.. image:: ../images/motororder-y6f-2d.png
-    :target: ../_images/motororder-y6f-2d.png
-    :scale: 35%
-
-Octocopter
-----------
-
-.. image:: ../images/motororder-octo-x-2d.png
-    :target: ../_images/motororder-octo-x-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-octo-plus-2d.png
-    :target: ../_images/motororder-octo-plus-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-octo-v-2d.png
-    :target: ../_images/motororder-octo-v-2d.png
-    :scale: 40%
-
-.. image:: ../images/motororder-octo-h-2d.png
-    :target: ../_images/motororder-octo-h-2d.png
-    :scale: 40%
-
-OctoQuad
---------
-
-.. image:: ../images/motororder-octoquad-x-2d.png
-    :target: ../_images/motororder-octoquad-x-2d.png
-    :scale: 35%
-
-.. image:: ../images/motororder-octoquad-plus-2d.png
-    :target: ../_images/motororder-octoquad-plus-2d.png
-    :scale: 35%
-
-.. image:: ../images/motororder-octoquad-v-2d.png
-    :target: ../_images/motororder-octoquad-v-2d.png
-    :scale: 35%
-
-.. image:: ../images/motororder-octoquad-h-2d.png
-    :target: ../_images/motororder-octoquad-h-2d.png
-    :scale: 35%
-
-DodecaHexacopter
```

Comment:
can you show the complete image output generated that will be inserted which includes the section titles and ordering?

Reply:
The short answer is yes to showing page output.

And yes, you can build this branch with absolutely no changes to the build  environment as it stands now. That was my goal, and it was met.

Here's a longer explanation of where I'm headed so you have fewer worries:

I was hard at work this weekend hacking up a way to pull this stuff down from `ArduPilot/ardupilot`, and I'm about to abandon that idea entirely. It's just too fragile, and I don't want to add unnecessary complexity.

Instead, I will update this PR to do the following:
- keep the page tracked and editable in its usual location
- take the image generation calls out of `update.py`
- place the script into `scripts` for manual execution
- add a page under `dev` describing its usage for updating the images and link it at the bottom of the "Connect Motors and ESCs" page.

I think that will satisfy everyone in the most amicable (and non-breaking) way.

Reply:
Having an image file(s) that can be generated from which I could cut and paste as needed when a new motor set is added would be very helpful, as well as leaving the wiki page layout,etc. still edited normally... would be great,thanks